integration/nodejs/express.mdx

```diff
@@ -5,67 +5,131 @@ icon: js
 public: true
 ---
 
-
-
 ## 1. Install dependencies
 
 <CodeGroup>
   ```npm npm
-  npm install @civic/auth cookie-parser
+  npm install @civic/auth cookie-parser cors
   ```
 
-  ```yarn yarn
-  yarn add @civic/auth cookie-parser
-  ```
+```yarn yarn
+yarn add @civic/auth cookie-parser cors
+```
 
-  ```pnpm pnpm
-  pnpm install @civic/auth cookie-parser
-  ```
+```pnpm pnpm
+pnpm install @civic/auth cookie-parser cors
+```
+
+```bun bun
+bun add @civic/auth cookie-parser cors
+```
 
-  ```bun bun
-  bun add @civic/auth cookie-parser
-  ```
 </CodeGroup>
 
 ## 2. Configure your App
 
-Your app will need the following configuration:
+### Minimal Configuration
+
+```ts
+const config = {
+  clientId: // Client ID from auth.civic.com
+  redirectUrl: 'https://your-backend.com/auth/callback' // change to your domain when deploying
+};
+```
+
+### Full Configuration (with logout and SPA support)
 
 ```ts
 const config = {
   clientId: // Client ID from auth.civic.com
-  redirectUrl: 'http://localhost:3000/auth/callback' // change to your domain when deploying,
-  postLogoutRedirectUrl: 'http://localhost:3000/' // The postLogoutRedirectUrl is the URL where the user will be redirected after successfully logging out from Civic's auth server.
+  redirectUrl: 'https://your-backend.com/auth/callback', // OAuth callback URL
+  postLogoutRedirectUrl: 'https://your-frontend.com/', // Where to redirect after logout
+  loginSuccessUrl: 'https://your-frontend.com/', // Optional: redirect SPAs back to frontend after auth
+  oauthServer: 'https://auth.civic.com/oauth' // Optional: OAuth server URL (for development/testing)
```

Comment:
I don't think we should put in oauthServer as an option here, even if it's available. We shouldn't encourage users to call dev or preprod endpoints